velox/exec/GroupingSet.cpp

```diff
@@ -423,8 +423,17 @@ void GroupingSet::initializeGlobalAggregation() {
   // requirements of all aggregate functions are satisfied.
 
   // Allocate space for the null and initialized flags.
+  int32_t numAggregates = aggregates_.size();
```

Comment:
Hi @waitinfuture , Cant land this because our linter is complaining about the types here, can you change the types : 
```
narrowing conversion from 'std::vector::size_type' (aka 'unsigned long') to signed type 'int32_t' (aka 'int') is implementation-defined
```